src/components/NewsList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '../styles/NewsList.css';
+
+const NewsList = props => {
+
+    const { articles } = props;
+
+    if (articles.length > 0) {
+        return (
+            <div className="News-Headline-Container">
+                <ul className="News-Headline-List">
+                    {articles.map((article, index) => {
+                        return (
+                            <li key={index} className="News-Headline-List-Items">
```

Comment:
@concodex93 In map functions, you can pass in an index variable which is perfect for using as the key I think because it will always be unique within the context.